packages/xcm-types/xcm-types.ts

```diff
@@ -69,15 +93,18 @@ type GenericAsset<Id, Fun> = {
   fun: Fun;
 };
 
+export type AssetV5 = GenericAsset<AssetIdV5, FungibilityV5>;
 export type AssetV4 = GenericAsset<AssetIdV4, FungibilityV4>;
 export type AssetV3 = GenericAsset<AssetIdV3, FungibilityV3>;
 export type AssetV2 = GenericAsset<AssetIdV2, FungibilityV2>;
 
 export type AssetId = AssetIdV4;
-export type AssetIdV4 = Location;
+export type AssetIdV4 = LocationV4;
+export type AssetIdV5 = AssetIdV4;
 export type AssetIdV3 = {concrete: LocationV3} | {abstract: Uint8Array};
 export type AssetIdV2 = {concrete: LocationV2} | {abstract: Uint8Array};
 
+// todo - should be a union, V5, or V4 (because V4 is wider than V5)
 export type Location = LocationV4;
```

Comment:
It should be just `LocationV5`. The latest XCM version is the library's default API.

The fields that got removed from NetworkId in V5 can be represented with the `ByGenesis` variant.

```suggestion
export type Location = LocationV5;
```